docs/source/interaction.rst

```diff
@@ -65,7 +65,7 @@ A minimal YAML formatted configuration for each of the Demo and BlueBrainNexus s
     """
    forge = KnowledgeGraphForge(configuration=config)
 
-See the `configuration documentation page section <https://nexus-forge.readthedocs.io/en/latest/configuration.html>`__ for a full list of forge configuration and see the `examples/notebooks/use-cases/ <https://github.com/BlueBrain/nexus-forge/tree/master/examples/notebooks/use-cases>`__ folder
+See the `configuration documentation page section <https://nexus-forge.readthedocs.io/en/v0.8.1/configuration.html>`__ for a full list of forge configuration and see the `examples/notebooks/use-cases/ <https://github.com/BlueBrain/nexus-forge/tree/master/examples/notebooks/use-cases>`__ folder
```

Comment:
If somebody hits https://nexus-forge.readthedocs.io/en/v0.8.0/interaction.html, it will then be directed to the latest config which might have changed compared to what it was in v0.8.0. Now it would be good to have the tag injected instead of manually edited